From 5588ef8e164248e90b22d9d6beb2493f302742f1 Mon Sep 17 00:00:00 2001
From: Richard Shann <richard@rshann.plus.com>
Date: Sun, 7 Sep 2025 13:13:44 +0100
Subject: [PATCH 1/4] Version 2.6.49

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 988a1d481..3c4afe6bc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,5 +1,5 @@
 dnl Process this file with autoconf to produce a configure script.
-AC_INIT([denemo], [2.6.48], [denemo-devel@gnu.org])
+AC_INIT([denemo], [2.6.49], [denemo-devel@gnu.org])
 AC_CONFIG_SRCDIR([src/core/main.c])
 AM_INIT_AUTOMAKE([tar-pax foreign dist-xz subdir-objects])
 dnl AC_CANONICAL_SYSTEM
-- 
2.51.0

