From 171bdfbe7b4347e0fbcfa0f0b2de9609702e76bc Mon Sep 17 00:00:00 2001
From: Federico Bruni <fede@inventati.org>
Date: Wed, 23 Mar 2022 22:55:28 +0100
Subject: [PATCH 2/2] denemo metainfo: add 2.6.0 release

---
 org.denemo.Denemo.appdata.xml.in | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/org.denemo.Denemo.appdata.xml.in b/org.denemo.Denemo.appdata.xml.in
index 393de9e46..935484a1b 100644
--- a/org.denemo.Denemo.appdata.xml.in
+++ b/org.denemo.Denemo.appdata.xml.in
@@ -46,5 +46,8 @@
     <keyword>MIDI</keyword>
     <keyword>Music</keyword>
  </keywords>
+ <releases>
+  <release type="stable" version="2.6.0" date="2022-03-11" />
+ </releases>
  <content_rating type="oars-1.1" />
 </application>
-- 
2.33.1

